src/KafkaFlow.Serializer.SchemaRegistry.ConfluentProtobuf/ConsumerConfigurationBuilderExtensions.cs

```diff
@@ -1,8 +1,7 @@
-﻿namespace KafkaFlow
+﻿namespace KafkaFlow.Serializer.SchemaRegistry
```

Comment:
please undo this change. The configuration builders should be in KafkaFlow root namespace to make it easier for the client application to use